web/package.json

```diff
@@ -56,19 +56,19 @@
     "d3-selection": "^1.3.0",
     "d3-transition": "^1.1.1",
     "deline": "^1.0.4",
-    "draft-js": "^0.10.5",
-    "draftjs-to-html": "^0.8.4",
+    "draft-js": "0.10.5",
+    "draftjs-to-html": "0.8.4",
     "formik": "^1.0.1",
     "history": "^4.7.2",
-    "html-to-draftjs": "^1.4.0",
+    "html-to-draftjs": "1.4.0",
     "i18n-js": "^3.0.3",
     "lodash.kebabcase": "^4.1.1",
     "markdown-it": "^8.4.1",
     "numeral": "^2.0.6",
     "prop-types": "^15.6.0",
     "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "react-draft-wysiwyg": "^1.10.7",
```

Comment:
Remove the version range specifier.  Let's lock this puppy (and the other draftjs-related deps) down.